packages/contentful--cra-template-create-contentful-app/template/gitignore

```diff
@@ -10,6 +10,7 @@
 
 # production
 /build
+/dist
```

Comment:
Why? Create React App always builds to `build`.

Reply:
Mmmh, it's not necessary for every fork of the project but in general `dist` is an output folder. We "needed" it for the tasks-app when we want to have the deploy step in [this PR](https://github.com/contentful/tasks-app/pull/33/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R61).

Reply:
```suggestion
```